lib/webvalve/version.rb

```diff
@@ -1,3 +1,3 @@
 module WebValve
-  VERSION = "1.2.0"
+  VERSION = "2.0"
```

Comment:
```suggestion
  VERSION = "2.0.0"
```

This might not break anything, but in some unrelated exploration of rubygem versions that I was doing, I found only a tiny handful that _didn't_ go all the way out to patch versions even if they're 0's. Rubygems doesn't expand the remaining 0's for you, and I'm not sure what implications it has elsewhere, other than breaking the tool I was trying to write...